src/Components/WeatherIcon/WeatherIcon.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const WeatherIcon = ({ icon, description }) => {
+  const imageLink = require(`../../assets/images/icons/${icon}.png`);
```

Comment:
no require inside component.

just do it like this.
Create ENUM for icons

```
const Icons = Object.freeze({
  SUN: 'sun',
  CLOUDY: 'cloud'
});

const WeatherIcon = ({description, iconName}) => (
  <img alt="Weather icon" src={`/assets/images/icons/${iconName},png`}/>
);
```

Consider also, If you have like 4-5 Diffrent icons it would be good idea to Create separate component for every icon.

Reply:
Basicly I can't do thac cuz name I'm taking from API, so it can't be static values

Reply:
but you can do just this

const WeatherIcon = ({description, iconName}) => (
  <img alt="Weather icon" src={`/assets/images/icons/${iconName},png`}/>
);

you shouldn't use require inside component